### DISCUSSION TITLE
How to change price's precision/decimal places?

### DISCUSSION THREAD

**Discussion by timocov** [tradingview/lightweight-charts#697] [Q&A] [answered]
By default lightweight-charts displays only 2 digit for any price, even if the price has more precision. How to change that so I can show more or less numbers after a comma?

**Comment by timocov** [comment, marked as answer]
Yes, it's possible. To change format of your series' data you need to override [price format](https://tradingview.github.io/lightweight-charts/docs/api/interfaces/SeriesOptionsCommon#priceformat) for your series by changing/applying options.

For example, if you'd like to show 6 digits after a comma you can use the following options:

```js
series.applyOptions({
    priceFormat: {
        type: 'price',
        precision: 6,
        minMove: 0.000001,
    },
});
```

Also, you can look at [this test case](https://github.com/tradingview/lightweight-charts/blob/b75839519282295ded1b8a6b8bae9a4686f45de1/tests/e2e/graphics/test-cases/initial-options/price-format.js) for more complete example.

**Reply by rusinov-artem** [reply]
link to documentation is broken (

**Reply by timocov** [reply]
@rusinov-artem thanks! I just updated the link to https://tradingview.github.io/lightweight-charts/docs/api/interfaces/SeriesOptionsCommon#priceformat. Also, you could switch to a tag of your version https://github.com/tradingview/lightweight-charts/blob/v3.7.0/docs/series-basics.md#price-format.

**Comment by Gilson401** [comment]
Hi. How to set zero decimal places correctly? If I use "precision: 0" this shows one decimal place.

**Reply by groehm** [reply]
add a minMove: 1   (something without decimal)

**Comment by MohsenTozandejani** [comment]
this.tvWidget.applyOverrides({
        'mainSeriesProperties.minTick': '1,1,false'
      });
